test/e2e/user_workload_monitoring_test.go

```diff
@@ -61,7 +63,8 @@ func TestUserWorkloadMonitoring(t *testing.T) {
 		{"create prometheus and alertmanager in user namespace", createPrometheusAlertmanagerInUserNamespace},
 		{"assert user workload metrics", assertUserWorkloadMetrics},
 		{"assert user workload rules", assertUserWorkloadRules},
-		{"assert tenancy model is enforced", assertTenancyForMetrics},
+		{"assert tenancy model is enforced for metrics", assertTenancyForMetrics},
+		{"assert tenancy model is enforced for rules", assertTenancyForRules},
```

Comment:
does it make sense to add another bullet point in https://issues.redhat.com/browse/MON-1170 to separate out the e2e tenancy test for user workload monitoring into its own e2e scenario?
(not an issue we have to address in this PR)

Reply:
Since tenancy tests require metrics and alerts in the first place, I would assume that they can be included in the user workload metric tests and alerts (respectively).